twolevel_dbm.py

```diff
@@ -233,6 +233,13 @@ def secondlevel(inputs, args, secondlevel=False):
                 f"antsApplyTransforms -d 3 -t [output/secondlevel/secondlevel_{subjectname}{i}0GenericAffine.mat,1] "
                 f"-r output/secondlevel/secondlevel_template0.nii.gz --verbose -o [output/compositewarps/secondlevel/{subjectname}_affine.nii.gz,1]")
 
+###############################################################################
+            # Register final model to common space
+            if args.resample_to_common_space and is_non_zero_file(args.resample_to_common_space):
+                commands.append(
+                    f"antsRegistrationSyN.sh -d 3 -f {args.resample_to_common_space} -m output/secondlevel/secondlevel_template0.nii.gz -o output/secondlevel/template0_common_space_")
```

Comment:
I'd like to expose the options for antsRegistrationSyN.sh as another block in the argparse code but the defaults are fine for now.